CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift

```diff
@@ -73,6 +80,37 @@ final class CameraPreviewViewController: UIViewController {
         previewView.addGestureRecognizer(tapGesture)
         return previewView
     }()
+
+    private let iPadCameraPaneWidth: CGFloat = 124
+    private lazy var iPadLandscapeConstraints: [NSLayoutConstraint] = [
+        cameraFrameView.topAnchor.constraint(equalTo: view.topAnchor, constant: 16),
+        cameraFrameView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
```

Comment:
[Suggestion] @vizaknaidavid Could you add some constant for the  numbers like '16', '1.414'?